cpu/stm32/stm32_info.mk

```diff
@@ -40,10 +40,10 @@ else ifneq (,$(filter $(CPU_FAM),g4 wb wl))
   CPU_CORE = cortex-m4
 else ifeq (f7,$(CPU_FAM))
   CPU_CORE = cortex-m7
-else ifneq (,$(filter $(CPU_FAM),g0 l0))
+else ifneq (,$(filter $(CPU_FAM),g0 l0 c0))
   CPU_CORE = cortex-m0plus
 else ifneq (,$(filter $(CPU_FAM),l5 u5))
   CPU_CORE = cortex-m33
 else
-  $(error Not supported CPU family: 'stm32$(CPU_FAM)')
+  $(error Not supported CPU family: 'stm32 $(CPU_FAM)')
```

Comment:
why the space

Reply:
Before editing this file at all, the error that was printed was

```
Not supported CPU family: 'stm32stm32c031c6'.
```

which I felt was unclear because I wasn't trying to add `stm32stm32c031c6`.  So I put the space in there temporarily just to confirm it was this error and not some prefix string I had duplicated somewhere.  I realized I needed two changes in this file, on lines 11 and 43.  The error as implemented is only really clear if you had made the edit on line 11, but forgotten the edit in the later half of the file.  Maybe the correct edit would have been this instead...

```
  $(error Not supported STM32 CPU family: '$(CPU_FAM)')
```

or adding `$(STM32_TYPE)` or `$(STM32_FAMILY)` to the message ?

Reply:
Looks like there is one `stm32` too much in the output then, just remove it instead of adding a space.

Reply:
Ok.  Related, this similar error message is in STM32's [cpu_conf.h Line #81](https://github.com/RIOT-OS/RIOT/blob/be1efe80c982c7efdd7ebf694087dd772d83147e/cpu/stm32/include/cpu_conf.h#L81C1-L82C1) with this message...  
```
#error Not supported CPU family
```
Are you okay with those being similar or should they be more distinct?

Reply:
removed prefixed STM32